### PR TITLE
Standardize code style

### PR DESCRIPTION
**Pull request by mzgoddard**
[Preview Tests](https://raw.githack.com/w3c/aria-at/mzgoddard-prettier/build/index.html)

Bring this repo closer in line with code style used in aria repos based on [style guide in w3c/aria-practices](https://github.com/w3c/aria-practices/wiki/Code-Guide#javascript-1).

- Install husky pre-commit hook to lint and fix code style changes
- Use eslint-plugin-prettier to run prettier as part of eslint and list prettier style issues in eslint's output

As part of the lint pass in this change, https://github.com/w3c/aria-at/pull/463 is essentially included in this PR.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "printWidth": 100
```
Comment (alflennik) [inline .prettierrc:3]: Should we add `"arrowParens": true` as recommended by the Airbnb style guide?

Comment (mzgoddard) [reply .prettierrc]: Added

**Review by alflennik — APPROVED** [review]
Thanks for quickly implementing that one note Z! Looks great. Hopefully we can get this merged relatively quickly to avoid the inevitable merge conflicts.

**Review by howard-e — APPROVED** [review]
Looks good @mzgoddard! This is a much needed change. I agree with having this quickly merged.

**Pull request merged**